### DISCUSSION TITLE
Changing behavior's name

### DISCUSSION THREAD

**Discussion by fabiomachadofonseca** [thelfer/tfel#410] [Q&A] [answered]
Hello,

I am using the behavior MericCailletaudSingleCrystalViscoPlasticityNumericalJacobian with Abaqus and I had this problem :

 _***ERROR: VALUE FOR PARAMETER NAME CANNOT BE MORE THAN 80 CHARACTERS
 LINE IMAGE: *Material, 
             name=ABAQUSBEHAVIOUR_MERICCAILLETAUDSINGLECRYSTALVISCOPLASTICITYNUM
             ERICALJACOBIAN_3D_LAYER1_

How can I solve it?

Thanks for your help!
Regards

**Comment by thelfer** [comment]
The `ABAQUS` name contains both the name of the library and the name of the behaviour. The first one can be changed by the @Library keyword (but you can also change it by hand by renaming the shared library) and the second one can be changed using the @Behaviour keyword.

**Reply by fabiomachadofonseca** [reply]
Thanks for you answer @thelfer. It's important to ensure consistency between the behavior name and its corresponding function in the integrator. In this case, to update the Behavior_SlipSystems function (that I forgot to change :)

**Reply by thelfer** [reply, marked as answer]
@fabiomachadofonseca don't hesitate if you have other questions !
